qcfractal/interface/tests/test_visualization.py

```diff
@@ -21,8 +21,9 @@ def live_fractal_or_skip():
     """Ensure Fractal live connection can be made"""
 
     try:
+        requests.get('https://api.qcarchive.molssi.org:443', json={}, timeout=5)
         return portal.FractalClient()
-    except requests.exceptions.ConnectionError:
+    except (requests.exceptions.ConnectionError, ConnectionRefusedError):
         return pytest.skip("Could not make a connection to central Fractal server")
 
```

Comment:
The visualization tests fail because `ReactionDataset._visualize` calls `ReactionDataset.get_history`. A few things need to be done:

1. `ReactionDataset._visualize` should call `ReactionDataset.get_values`.
2. `ReactionDataset.get_history` should call  `ReactionDataset.get_values` instead of the non-existent `ReactionDataset._get_history`.
3. There should be a test for `ReactionDataset.get_history` added, which can be removed when `ReactionDataset.get_history` is deprecated. (I think what happened is the `ReactionDataset.get_history` tests were rewritten to use `ReactionDataset.get_values`.)